python/grass/pygrass/modules/interface/flag.py

```diff
@@ -96,7 +96,7 @@ def __nonzero__(self):
     def __doc__(self):
         """Return a documentation string, something like:
 
-        {name}: {default}, suppress required {supress}
+        {name}: {default}, suppress required {suppress}
```

Comment:
```suggestion
        {name}: {default}, suppress required {supress}
```

Reply:
Looks like there is one at 
https://github.com/OSGeo/grass/blob/43215aec294a8f79063776f7c3356d38a8930b57/python/grass/pygrass/modules/interface/flag.py#L118

Reply:
Ok, if you can fix the variable name there, it might work, as it looks like a template to create docstrings of functions programmatically. If there are other places in the pygrass/modules/interface they should be consistent with each other too.